src/Package/Impl/Utilities/ProjectUtilities.cs

```diff
@@ -56,10 +56,20 @@ public static class ProjectUtilities {
         public static EnvDTE.Project GetActiveProject() {
             DTE dte = VsAppShell.Current.GetGlobalService<DTE>();
             if (dte.Solution.Projects.Count > 0) {
-                var projects = (Array)dte.ActiveSolutionProjects;
-                if (projects != null && projects.Length == 1) {
-                    return (EnvDTE.Project)projects.GetValue(0);
-                }
+                try {
+                    if (dte.Solution != null) {
```

Comment:
Given that we have accessed `dte.Solution` three lines above, this seems redundant (or else that line should also be inside `if`).